realm/realm-library/src/main/java/io/realm/log/RealmLogger.java

```diff
@@ -22,6 +22,7 @@
  * Interface for custom loggers that can be registered at {@link RealmLog#add(RealmLogger)}.
  * The different log levels are described in {@link LogLevel}.
  */
+@KeepMember
```

Comment:
yes.
I'd like to achieve only requiring to add the annotation to methods, but I couldn't find the way.